proposals/artifacts.md

```diff
@@ -0,0 +1,68 @@
+# OCI artifact project proposal
+
+## Abstract
+Container registries, implementing the [OCI distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers use cloud provider implementations, vendor implementations, and instances of the open source implementation of docker distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional value atop their registry implementations from security to productivity features. 
+
+Applications and services typically require additional artifacts to deploy and manage container images, including [helm](https://helm.sh) charts for deployment and [Open Policy Agent (OPA) bundles](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
+
+Utilizing the [OCI manifest][image-manifest] and [OCI index][image-index] definitions, new artifact types can be stored and served using the [OCI distribution-spec][distribution-spec] without changing the actual distribution spec. This repository will provide a reference for artifact authors and registry implementors for supporting these new artifact types themselves with the existing implementations of distribution.
+
+By providing support for OCI artifact types over OCI distributions, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS). 
+
+## Proposal
+Under the http://github.com/opencontainers organization:
+
+- Create a new **artifacts** repository, named http://github.com/opencontainers/artifacts
+- Update the [OCI distribution][distribution-spec] to generically reference [OCI manifest][image-manifest] and [OCI index][image-index], with image as one of many artifact types it can store. 
```

Comment:
s/the [OCI distribution]/the [OCI distribution spec]/
s/with image/[OCI image][image-spec]/